src/compiler/types.ts

```diff
@@ -1233,6 +1233,10 @@ namespace ts {
         isBracketed: boolean;
     }
 
+    export interface PackageDescriptor extends Scope {
+        packageFile: string;

```

Comment:
Can you add a comment? Specifically mention that this can be a `package.json` or `index.d.ts`.